### ISSUE TITLE
Rollover stops without any error in logs

### ISSUE THREAD

**Issue by echerniak** [elastic/curator#1538]
### Expected Behavior
Rollover should succeed.

### Actual Behavior
Rollover doesn't complete and no errors appear in logs.

### Steps to Reproduce the Problem
1. Create 3 indexes:

```bash
  curl -XPUT "http://elasticsearch:9200/index-2020.04.14-000001" -H 'Content-Type: application/json' -d'{}'
  curl -XPUT "http://elasticsearch:9200/index-2020.04.14-000002" -H 'Content-Type: application/json' -d'{}'
  curl -XPUT "http://elasticsearch:9200/index-2020.04.14-000003" -H 'Content-Type: application/json' -d'{}'
  ```

2. Add all indexes into an alias, but leave **is_write_index** blank for one of them:

   ```bash
   curl -XPOST "http://elasticsearch:9200/_aliases" -H 'Content-Type: application/json' -d'{  "actions": [    {      "add": {        "index": "index-2020.04.14-000001",        "alias": "index_alias"      }    },    {      "add": {        "index": "index-2020.04.14-000002",        "alias": "index_alias",        "is_write_index": false      }    },    {      "add": {        "index": "index-2020.04.14-000003",        "alias": "index_alias",        "is_write_index": true      }    }  ]}'
   ```

3. Run Curator with the following actions.yml:

   ```yaml
   actions:
      1:
       action: rollover
       description: Rollover the index associated with alias 'index_alias', which should be in the form of index-YYYY.MM.DD-000001.
       options:
         name: index_alias
         conditions:
           max_age: 1m
         continue_if_exception: False
   ```

4. The last record in Curator log is:

   ```
   2020-04-15 16:09:34,639 DEBUG              elasticsearch    log_request_success:125  < {"index-2020.04.14-000002":{"aliases":{"index_alias":{"is_write_index":false}}},"index-2020.04.14-000001":{"aliases":{"index_alias":{}}},"index-2020.04.14-000003":{"aliases":{"index_alias":{"is_write_index":true}}}}
   ```

   There is no error in the logs, which makes it difficult to identify the root problem.

### Specifications

  - Version: Curator 5.8.1, Elasticsearch 7.6.1
  - Platform: Red Hat Enterprise Linux Server release 7.6 (Maipo)

## Detailed Description
It seems like there are 2 problems: 

1. [get_write_index](https://github.com/elastic/curator/blob/b131026c6e7de8d98e024ee858c1ccadadd6598a/curator/utils.py#L2065) procedure doesn't expect to receive index with alias without **is_write_index** parameter. 

   It seems incorrect to me, because **is_write_index** is an [optional](https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-aliases.html#indices-aliases-api-request-body) parameter and this command still works fine:

   ```bash
   curl -XPOST "http://elasticsearch:9200/index_alias/_rollover/"
   ```

   

2. After get_write_index procedure receives index with alias without **is_write_index** parameter, it raises an exception "Invalid alias: is_write_index not found in 1 to many alias". But this text is getting missed due to 

   ```python
   except Exception as e:
       if 'ASCII' in str(e):
           print('{0}'.format(e))
           print(__doc__)
   ```

   [condition](https://github.com/elastic/curator/blob/b131026c6e7de8d98e024ee858c1ccadadd6598a/run_curator.py#L36).

**Comment by untergeek** [comment]
I can perhaps fix this, but you should also know that you should not rollover an alias that is associated with indices that do not have `is_write_index` set to either `true` or `false`. This is now default behavior when rollover aliases and indices are created. After rollover, the new index will have `is_write_index` set to `true`, and the old index will still have `is_write_index` present, but set to `false`. Curator deliberately will fail if you try to rollover an alias that does not have `is_write_index` set to either `true` or `false` for this very reason.

If you're trying to keep an alias set to point to all indices in a group that is older and never had `is_write_index`, you should add that setting to those indices. Fortunately, Curator ought to be able to help with that.

**Comment by echerniak** [comment]
I agree about setting `is_write_index` via curator and do it this way, thanks! However, I am more confused by the lack of errors in Curator logs. Is it possible to make them more informative when this situation occurs?

**Comment by untergeek** [comment]
I will still look into that.

**Comment by rusnw** [comment]
Is the project still alive? We are experiencing big problems with the informativeness of the curator’s logs not only for this issue. One gets the feeling that the project is more likely to have died.

**Comment by untergeek** [comment]
Not dead, but barely on life support. I just don't have a lot of time to spend on it.
